include/swift/AST/Types.h

```diff
@@ -2713,7 +2711,7 @@ enum class ParameterConvention {
   Direct_Guaranteed,
 };
 
-inline bool isIndirectParameter(ParameterConvention conv) {
+inline bool isIndirectFormalParameter(ParameterConvention conv) {
```

Comment:
Is it possible to make ParameterConvention a struct enum? Then this could be a method!

I am fine with this being in a different commit/PR.